style.css

```diff
@@ -0,0 +1,675 @@
+  font-size: 20px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+  max-width: 500px;
+}
+
+.logo-section .logo {
+  margin-top: 32px;
+  border: 1px solid #2c2c2c;
+  max-width: 370px;
+  margin-bottom: 20px;
+}
+
+.past-summits-sections {
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 25px;
+  background-color: #fff;
+}
+
+.past-summits-sections .past-summits-heading {
+  margin-bottom: 15px;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  color: #2c2c2c;
+  line-height: 28px;
+}
+
+.past-summits-sections .past-summits-desc {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 21px;
+  color: #838383;
+  text-align: center;
+  margin-top: 15px;
+}
+
+.past-summits-sections .past-summits-images-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-self: center;
+  margin-top: 15px;
+}
+
+.past-summits-images-container .img-container .img-content-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.past-summits-images-container .img-container .img-content-container .summit-year {
+  font-family: "Lato", sans-serif;
+  font-size: 15px;
+  text-shadow: 1px 1px #363434;
+  margin-bottom: 10px;
+}
+
+.past-summits-images-container .img-container .img-content-container .summit-desc {
+  font-family: "Lato", sans-serif;
+  font-size: 8px;
+  text-shadow: 1px 2px #363434;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: #414246;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  align-items: center;
+  overflow: auto;
+}
+
+.partners .partners-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.partners .partner-icons-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+}
+
+.partners .partner-icons-list .partner-icons-list-child {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  margin-top: 30px;
+}
+
+.footer {
+  background-color: #fff;
+  padding: 60px 80px;
+  display: flex;
+  flex-direction: row;
+}
+
+.footer .copyright {
+  text-align: left;
+  height: 24px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 24px;
+  margin-left: 20px;
+  margin-top: 30px;
+}
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    background-color: white;
+  }
+
+  #toggle-icon {
+    display: none;
+  }
+
+  .navbar .logo-container {
+    margin-left: 15px;
+    display: block;
+  }
+
+  .navbar .logo-container img {
+    width: 50px;
+  }
+
+  .links-container ul {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-top: 20px;
+    padding-right: 70px;
+  }
+
+  .links-container ul li {
+    margin-left: 45px;
+    cursor: pointer;
+  }
```

Comment:
- [ ] KIndly make sure your navigation links have the same typography (color) as the one on the provided Behance design. Kindly make sure the style for the active link is the same as the provided design.


    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/225589291-70eb80c9-eb6f-4417-8e8e-32fb000027b6.png)

    - ### BEHANCE DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/225589379-fe26a159-38ea-4300-99f9-de860472a9b1.png)